src/main/kotlin/no/nav/tilleggsstonader/sak/behandling/dto/BehandlingDto.kt

```diff
@@ -39,11 +40,13 @@ data class BehandlingDto(
     val henlagtBegrunnelse: String?,
     val revurderFra: LocalDate?,
     val nyeOpplysningerMetadata: NyeOpplysningerMetadata?,
+    val tilordnetSaksbehandler: AnsvarligSaksbehandlerDto?,
```

Comment:
Er det ryddig om det enten heter tilordnetSksbehandler eller ansvarligSaksbehandler hele veien?
(Ser det er flere av disse i koden så bør nok bestemme oss for det enne eller det andre)
```suggestion
    val ansvarligSaksbehandler: AnsvarligSaksbehandlerDto?,
```

Reply:
Ja hundre prosent, her er det jeg som sslurver